src/index.html

```diff
@@ -1,12 +1,384 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="index.html" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+          class="header__logo-img"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="index.html">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="index.html">Samsung</a>
+          </li>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="index.html">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="index.html">Samsung</a>
          </li>
```
add spaces between blocks of code